picasso/src/main/java/com/squareup/picasso3/RequestHandler.java

```diff
@@ -81,6 +100,18 @@ public boolean hasBitmap() {
       return bitmap != null;
     }
 
+    /**
+     * The loaded {@link Drawable}.
+     * Mutually exclusive with {@link #getBitmap()}.
+     */
+    @Nullable public Drawable getDrawable() {
```

Comment:
Should these throw if the wrong one is called?  Then you wouldn't need nullable.

Reply:
At the moment, no, since it's because that neither a non-null Drawable nor a non-null Bitmap is returned.  Definitely something to clean up later.